django_mongodb/query.py

```diff
@@ -319,6 +319,36 @@ def raw_mql(self, raw_query, params=(), translations=None, using=None):
 
 
 class MongoRawQuery(RawQuery):
+
+
+    def __init__(self, sql, using, model, params=()):
+        self.params = params
```

Comment:
You don't need to repeat the superclass implementation.
```
super().__init__(...)
self.model = model
```

Reply:
cfd2ace56d0de86a2f17c6792ec8796acbd1a01f